src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -100,7 +113,7 @@ private void addRandomUsers() {
     Collections.shuffle(randomPasswords);
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), randomPasswords.get(i), Instant.now());
+      User user = new User(UUID.randomUUID(), randomUsernames.get(i), randomPasswords.get(i), Instant.now(), (int) (Math.random()*50));
```

Comment:
Is the user just given a random number of messages for their stat? Should you instead make this 0 and add to the user's number of messages in addRandomMessages to make it more accurate?

Reply:
yeah right. I was trying to give it random numbers so i get different data for the word cloud, but you're right it should be counted when the messages are assigned.